src/components/App/index.js

```diff
@@ -69,12 +68,13 @@ export const App = ({ options }) => {
   }
 
   let nextQuestion = {};
-  const gameOverResults = [];
+  let gameOverResults = [];
   let level = 0;
   let quiz = {};
   const computerMind = new ComputerMind();
 
   async function play(gameMode) {
+    gameOverResults = [];
```

Comment:
jak już to masz zadeklarowane w 71 linijce to potrzebujesz dodatkowo nadpisywać tutaj?

Reply:
tak, żeby się resetowało między grami